### ISSUE TITLE
Filename uses Windows reserved characters

### ISSUE THREAD

**Issue by edwinstauthamer** [TeamMsgExtractor/msg-extractor#205]
**Bug Metadata**
* Version of extract_msg: [0.28.7]
* Your python version: Python [3.9.6]
* How did you launch extract_msg?
  - [ X] My command line or
  - [ ] I used the extract_msg package

**Describe the bug**
When using the subject of an e-mail that is an attachment of an e-mail that you want to extract, as the name of the subfolder, the program cannot extract the e-mail/create the subfolder when there are Windows FS reserved characters in the subject. 

[ If applicable ]
**What code did you use or can we use to reproduce this error?
I added a "hack" in message.py to overcome this issue. I replace the reserved characters with "_". 
I also cut of the name of the folder/file to 60 chars. 

```
if fileName.find("Y:") == -1:
fileName = fileName.replace(':','_').replace('\\','_').replace('/','_').replace('*','_').replace('?','_').replace('"','_').replace('<','_').replace('>','_').replace('|','_')
fileName = fileName[0:60]
```

**Comment by TheElementalOfDestruction** [comment]
A fix has been added already to the next version, the one in progress. 0.29.0 will fix many issues related to saving because it redoes the whole system.

**Comment by TheElementalOfDestruction** [comment]
Should now be fixed in 0.29.0. Let me know if there are issues.
